src/App.jsx

```diff
@@ -1,7 +1,11 @@
+import EmailList from "./components/EmailList"
+import Nav from "./components/Nav"
+
 function App() {
   return (
     <div>
-      <h1 className="text-3xl font-bold underline text-red-500">Hello, Dragons</h1>
+      <Nav />
```

Comment:
Really at this point you need some routing - as I mentioned in my comment on your dropdown, the 'inbox' button needs to link to the homepage, which technically doesn't currently exist since you don't have a route setup